substrate/frame/scheduler/src/lib.rs

```diff
@@ -295,10 +315,19 @@ pub mod pallet {
 			id: Option<TaskName>,
 			result: DispatchResult,
 		},
+		/// Set a retry configuration for some task.
+		RetrySet {
+			task: TaskAddress<BlockNumberFor<T>>,
+			id: Option<TaskName>,
+			period: BlockNumberFor<T>,
```

Comment:
It does not contain the `total_retries`?

Reply:
I didn't include it because I tried to keep the event lean and I thought the period is the most important aspect of it.

Reply:
Yea am also often not sure what all to include in events... but i think in this case its fine since it should only rarely be used.

Reply:
Added the retry count.